src/sentry/tasks/post_process.py

```diff
@@ -591,6 +592,18 @@ def get_event_raise_exception() -> Event:
         for job in group_jobs:
             run_post_process_job(job)
 
+        # TODO: Should we even emit this for reprocessing?
+        if start_time := event.data.get("received"):
```

Comment:
One idea is to figure out if the `received` timestamp is being bumped when you trigger reprocessing.
If not, then this metric would be meaningless, as it can even be a couple of weeks, as in: the time you triggered reprocessing.

Reply:
If I understand you correctly: If reprocessing doesn't refresh the `received` timestamp, emitting this metric for reprocessing would be pointless because the timestamp would be wildly out of date. Is that what you mean?